src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,7 +33,7 @@ class AssetsInstallCommand extends ContainerAwareCommand
     protected function configure()
     {
         $this
-            ->setName('sylius:assets:install')
+            ->setName('sylius:theme:assets:install')
```

Comment:
I just added `sylius:`**`theme`**`:assets:install`. It requires to use another command, but I think we can improve the DX later.